web/src/features/panels/zone/EstimationCard.cy.tsx

```diff
@@ -67,10 +79,10 @@ describe('EstimationCard with known estimation method', () => {
   });
 
   it('Toggles when collapse button is clicked', () => {
-    cy.get('[data-test-id="collapse-up"]').should('not.exist');
-    cy.get('[data-test-id="collapse-down"]').should('exist');
-    cy.get('[data-test-id="body-text"]').should('not.exist');
-    cy.get('[data-test-id="methodology-link"]').should('not.exist');
+    cy.get('[data-test-id="collapse-up"]').should('exist');
+    cy.get('[data-test-id="collapse-down"]').should('not.exist');
+    cy.get('[data-test-id="body-text"]').should('exist');
+    cy.get('[data-test-id="methodology-link"]').should('exist');
```

Comment:
Since the state is now persisted using an atom these become more or less inverted as it's clicked in the previous test.

I was hoping to find a easy solution to isolate them but I don't think my solution is fully isolating them.

Reply:
The estimation card was closed by default before but in `ResetEstimationCard` you set it to be open by default so this gets inverted. I think we might want to set `setIsCollapsed(true);` in `ResetEstimationCard` to get the same results as before. But for the outage card it was open by default so we might need to make another one for that

Reply:
Ah, I'll double check.

Reply:
Not sure where you mean I set it to open by default here?
I set it to undefined which means it should use the default values provided to the card type, either open or closed.

Reply:
Ahh okay I understand, but I don't think the solution to this is to change the test. I will try to figure out another way to do it:)